ci/ci.sh

```diff
@@ -83,7 +83,19 @@ esac
     --host="$HOST" $EXTRAFLAGS
 
 # We have set "-j<n>" in MAKEFLAGS.
-make
+build_exit_code=0
+make || build_exit_code=$?
```

Comment:
nit: This should be simpler
```suggestion
make
build_exit_code=$?
```

Reply:
> nit: This should be simpler

Doesn't `set -eux` in the top of the script force exiting when `make` fails?